src/event/cmap.rs

```diff
@@ -117,6 +117,9 @@ pub enum ConnectionClosedReason {
     /// An error occurred while using the connection.
     Error,
 
+    /// The connection was dropped during read or write
```

Comment:
nitpick, but can we add a period to the end of this doctring?

Reply:
Added